packages/light-apps/src/App.tsx

```diff
@@ -3,6 +3,7 @@
 // of the Apache-2.0 license. See the LICENSE file for details.
 
 import 'semantic-ui-css/semantic.min.css';
+import 'tachyons/css/tachyons.css';
```

Comment:
Similar to above: light-apps imports ui-components which import tachyons. I would remove this line, and keep the one in ui-components `globalStyle.ts`: https://github.com/paritytech/substrate-light-ui/pull/844/files#diff-f3b1dd94491a6c0092189efd048dbd1aR5.

Actually, i think it would be cleaner to:
- move `tachyons`, `tachyons-components` and `semantic-ui-css` dependencies away from `light-apps`, and move them into `ui-components`'s package.json
- in `globalStyle.ts`, where you did ` import 'tachyons/css/tachyons.min.css'; ` also add `import 'semantic-ui-css/semantic.min.css';` (so that we remove the semantic.min.css here)
- in `light-apps` package.json, we only have `ui-components` as dependency